### ISSUE TITLE
Make 3D evaluation

### ISSUE THREAD

**Issue by madhubabuv** [mrharicot/monodepth#104]
Hello,

How did you evaluate the model with make 3D? If it is possible please share the code.

**Comment by mrharicot** [comment]
Hi,
I am afraid I don't have the code for the Make3D evaluation anymore.

**Comment by madhubabuv** [comment]
I have been trying to replicate your make 3D dataset results but my code is not at all working. Can you please guide me how can I evaluate the model on that dataset?

**Comment by mtngld** [comment]
Can you share how did you crop the make3d images? And how did you align it with the depth image?

The make3d rgb images are of size 1704x2272, you write in your paper that you want to maintain the aspect ratio as in cityscapes which is 2:1 (images size is 2048x1024) - so I assume you take a center crop of the make3d rgb images to size 1704x852?

The make3d depth data is (55x305x4), I am assuming you used the third channel `Projective Depths in meters`  (https://cs.stanford.edu/group/reconstruction3d/Readme) - did you resize it to the 1704x2272 and later applied the same crop as for the RGB images?

Thank you!

**Comment by mrharicot** [comment]
Hi,
If I remember correctly, I cropped the top and bottom parts of the input images so that the ratio matched the one from the cropped (to remove the car hood) cityscapes images which is 2.5, the width was kept as is, then resized to 256x512 to feed to the neural net.

As for the ground truth depth I think I cropped the top and bottom as well and reshaped our prediction to match the cropped ground truth.

I hope this helps!
